lib/astc_encode.cpp

```diff
@@ -61,6 +61,8 @@ pthread_join(pthread_t thread, void** value) {
     return 0;
 }
 
+#elif defined(_WIN32) && defined(WIN32_HAS_PTHREADS)
```

Comment:
Change this line to  just `#else`

Reply:
Thank you for your kind guidance.
I had push a new RP please check it.
✧(≖ ◡ ≖ )